CHANGELOG.md

```diff
@@ -1021,7 +1021,7 @@ A big thanks to the 17 contributors who made this release possible. Here are som
 - &#8203;<!-- 25 -->[core] Update playwright docker to match the specified version (#31236) @siriwatknp
 - &#8203;<!-- 24 -->[core] Remove parallel on buildTypes (#31189) @siriwatknp
 - &#8203;<!-- 23 -->[core] Fix propTypes generation for optional any props (#31141) @m4theushw
-- &#8203;<!-- 04 -->[Typescript] Remove variants deprecation (#31239) @siriwatknp
+- &#8203;<!-- 04 -->[typescript] Remove variants deprecation (#31239) @siriwatknp
```

Comment:
the label is lower case

<img width="139" alt="Screenshot 2022-06-11 at 15 04 50" src="https://user-images.githubusercontent.com/3165635/173189155-886c5aa7-0e8e-4361-ac8b-f4590237c7cb.png">